conans/model/manifest.py

```diff
@@ -120,6 +119,9 @@ def create(cls, folder, exports_sources_folder=None):
 
         file_dict = {}
         for name, filepath in files.items():
+            if os.path.islink(filepath):
```

Comment:
I think it is more consistent to have them in the manifest, but with a 0 checksum or something like that.

Reply:
Sure, I forgot to remove this, no need to change anything here